bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/ChannelTypeConverter.java

```diff
@@ -142,6 +173,7 @@ protected ChannelTypeXmlResult unmarshalType(HierarchicalStreamReader reader, Un
         String description = super.readDescription(nodeIterator);
         String category = readCategory(nodeIterator);
         Set<String> tags = readTags(nodeIterator);
+        List<CommandOption> commandOptions = readCommandOptions(nodeIterator);
```

Comment:
Even though we do not yet have anything besides the options - wouldn't it be more symmetrical to have a "commandDescription" next to the state and event descriptions?